plugins/single_plugins/zoom.cpp

```diff
@@ -39,6 +40,9 @@ class wayfire_zoom_screen : public wayfire_plugin_t
 
             duration = wf_duration(smoothing_duration);
             duration.start(1, 1); // so that the first value we get is correct
+
+            GetTuple(x, y, output->get_cursor_position());
+            last_cursor_position = wf_point{x, y};
```

Comment:
You should still use `wlr_box_closest_point` here, nothing guarantees that the cursor will be on the output (since an output is focused when you click on it)

Reply:
I don't really understand what you mean. Do you want the zoom area to move even though the pointer is not on the output?

Reply:
Okay, maybe we need another check. What I mean is this: the scroll binding will be sent to the focused output, not the output that the mouse is on.

Reply:
Yes, that is how it currently works. How should it work instead?

Reply:
I updated to use `wlr_box_closest_point` and it still works the same way except the zoom area moves even when the cursor is not on the output. Is this what you wanted?